evkmimx8mm/driver_examples/ecspi/interrupt_b2b_transfer/master/pin_mux.c

```diff
@@ -35,12 +35,13 @@ processor_version: 0.0.0
  * END ****************************************************************************************************************/
 void BOARD_InitBootPins(void)
 {
+    BOARD_InitPins();
 }
 
 /*
  * TEXT BELOW IS USED AS SETTING FOR TOOLS *************************************
 BOARD_InitPins:
-- options: {callFromInitBoot: 'false', coreID: m4}
+- options: {callFromInitBoot: 'true', prefix: BOARD_, coreID: cm4}
 - pin_list:
```

Comment:
No prefix configuration is needed, should be  "- options: {callFromInitBoot: 'true', coreID: cm4}", same issue in other files.

Reply:
fixed